topics/statistics/tutorials/hyperdimensional_computing/tutorial.md

```diff
@@ -0,0 +1,147 @@
+>
+> 3. Rename the files {% icon galaxy-pencil %}
+>    - The names of files imported in the previous step are the full URL to those files;
+>    - Let's change their name to something a little clearer like `RA__ThomasAM__species.csv` and `BIN__ThomasAM__species.csv`.
+>
+>    {% snippet faqs/galaxy/datasets_rename.md %}
+{: .hands_on}
+
+# Build a classification model
+
+Hyperdimensional computing is an emerging brain-inspired computing paradigm that deals with vectors in a high-dimensional space. Every kind of information is thus encoded into random 10,000-length vectors (usually binary or bipolar) that are combined together to represent complex concepts. The length of the vectors is usually in the order of 10,000 to guarantee their quasi-orthogonality.
+
+There are usually three types of arithmetic operators that can be applied to combine vectors: bundling, binding, and permutation. They all have unique properties that must be taken into account when used ({% cite kanerva2009hyperdimensional %}).
+
+For what concerns the bundling operator: (i) the resulting vector is similar to the input vectors, (ii) the more vectors are involved in bundling, the harder it is to determine the component vectors, (iii) if several copies of any vector are included in bundling, the resulting vector is closer to the dominant vector than to the other components.
+
+The binding operation is instead: (i) invertible (unbinding), (ii) it distributes over bundling, (iii) it preserves the distance, (iv) and the resulting vector is dissimilar to the input vectors.
+
+Finally, the permutation operation is: (i) invertible, (ii) it distributes over bundling and any elementwise operation, (iii) it rpeserves the distance, and (iv) the resulting vector is dissimilar to the input vectors.
+
+The set of vectors and arithmetic operators used for combining vectors is called vector-symbolic architecture.
+
+`chopin2` implements a supervised classification model that works by encoding every observation in a dataset into high-dimensional vectors by combining values under their features. The model is built by collapsing the vector representation of the observations in the training set into a vector for each class of observations (e.g., in the case of the datasets retrieved in the previous step, classes are `CRC` and `control`). The classification model os then tested by computing the inner product between the vector representations of the observations in the test set against the two classes vectors. Vectors are thus classified based on their closeness to the classes.
+
+> <hands-on-title>Build a classification model with `chopin2`</hands-on-title>
+> **chopin2** {% icon tool %}: Run **chopin2** with the following configuration:
```

Comment:
You can annotate tools with Galaxy tool IDs. This then will yield of all kind of cool integrations.

Something like `{% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.11+galaxy0) %}`